### PR TITLE
[release-4.7] Convert into Cosa manifest repo

### PR DESCRIPTION
**Pull request by vrutkovs**
Build OKD image via cosa similar to master. This should resolve a lot of outstanding issues, like missing packages or disabled services etc.

Requires https://github.com/openshift/release/pull/19102 to pass.

Closes https://github.com/openshift/okd/issues/668

### CODE REVIEW

File: okd-rpms.repo [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[okd-rpms]
+name=OKD RPMs
+failovermethod=priority
```
Comment (LorbusChris) [inline okd-rpms.repo:3]: the `failovermethod` keyword is deprecated (see also: https://github.com/coreos/fedora-coreos-config/pull/1048)

Comment (vrutkovs) [reply okd-rpms.repo:3]: Filed https://github.com/openshift/okd-machine-os/issues/137

**Comment by vrutkovs** [comment]
/hold

Testing stable updates to this version

**Pull request merged**